networkx/drawing/nx_pylab.py

```diff
@@ -583,45 +585,29 @@ def draw_networkx_edges(G, pos,
     arrow_collection = None
 
     if G.is_directed() and arrows:
-
+    
         # a directed graph hack
         # draw thick line segments at head end of edge
         # waiting for someone else to implement arrows that will work
         arrow_colors = edge_colors
-        a_pos = []
         p = 1.0-0.25  # make head segment 25 percent of edge length
-        for src, dst in edge_pos:
+        for i, (src, dst) in enumerate(edge_pos):
             x1, y1 = src
             x2, y2 = dst
             dx = x2-x1   # x offset
             dy = y2-y1   # y offset
-            d = numpy.sqrt(float(dx**2 + dy**2))  # length of edge
```

Comment:
Careful. When this line (defining `d`) disappeared, your builds quit passing in Travis.